frontend/packages/dev-console/src/components/monitoring/overview/MonitoringTab.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { OverviewItem } from '@console/shared';
+import MonitoringMetricsSection from './MonitoringMetricsSection';
+
+type MonitoringTabProps = {
```

Comment:
```suggestion
export interface MonitoringTabProps = {
```

Reply:
@debsmita1 Don't need to export component props. Also `type` and `interface` are roughly the same thing (rather indifferent from component props point of view at least).

`React.ComponentProps<typeof MyComponent>` will get you the props out of a component without needing to export the props by themselves.